include/dfu/dfu_target.h

```diff
@@ -42,6 +42,7 @@ struct dfu_target {
 	int (*write)(const void *const buf, size_t len);
 	int (*done)(bool successful);
 	int (*schedule_update)(int img_num);
+	int (*reset)();
```

Comment:
Why do we call it "reset" - to me it seems like it only deletes the staging area for the updates. Would `delete`, `clear`, `erase` or something along those lines be more descriptive?

Reply:
This is equivalent of `dfu_target_reset()` like all other functions. It would be weird to map the functions otherwise
* target_write -> write()
* target_done -> done()
* target_schedule_update -> schedule_update()
* target_reset -> erase()    **This would be weird**

Reply:
I agree, I didn't notice that it was mapping to the existing dfu_target API.